packages/cli-platform-ios/src/tools/pods.ts

```diff
@@ -0,0 +1,98 @@
+import path from 'path';
+import fs from 'fs-extra';
+import {createHash} from 'crypto';
+import chalk from 'chalk';
+import {
+  CLIError,
+  cacheManager,
+  getLoader,
+} from '@react-native-community/cli-tools';
+import installPods from './installPods';
+import findPodfilePath from '../config/findPodfilePath';
+import {
+  DependencyConfig,
+  IOSDependencyConfig,
+} from '@react-native-community/cli-types';
+
+interface ResolvePodsOptions {
+  forceInstall?: boolean;
+}
+
+interface NativeDependencies {
+  [key: string]: DependencyConfig;
+}
+
+export function getPackageJson(root: string) {
+  return require(path.join(root, 'package.json'));
```

Comment:
shouldn't we try/catch here of from the caller? btw, Android introduced similar logic for resolving package.json in this PR: https://github.com/react-native-community/cli/pull/1635 maybe it's worth to share the implementation